spec/integration/grape_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+if defined?(Grape)
+  require 'elastic_apm/grape'
+  RSpec.describe 'Grape integration', :mock_intake do
+    include Rack::Test::Methods
+
+    let(:app) { GrapeTestApp }
+
+    before(:all) do
+      class GrapeTestApp < ::Grape::API
+        use ElasticAPM::Middleware
+
+        get :pingpong do
+          { message: 'Hello' }
+        end
+
+        resource :statuses do
+          desc 'Return a status.'
+          params do
+            requires :id, type: Integer, desc: 'Status id.'
+          end
+          route_param :id do
+            get do
+              { status: params[:id] }
+            end
+          end
+        end
+      end
+
+      config = { api_request_time: '100ms',
+                 capture_body: 'all',
```

Comment:
I don't think this is necessary unless we want to test that we actually capture the body